app/controllers/users_controller.rb

```diff
@@ -0,0 +1,58 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+    render json: @users
+  end
+
+  def show
+    if @user = User.find_by(id: params[:id])
+      render json: @user
+    else
+      render json: 'User not found'
```

Comment:
Read about I18n en.yml. Messages should be part of en.yml

Reply:
Updated.